langkit/templates/memoization_ada.mako

```diff
@@ -87,8 +87,7 @@ function Find_Memoized_Value
 procedure Add_Memoized_Value
```

Comment:
Typo in the commit message "exceution"

Reply:
Also I think this would warrant a change entry in the LAL repo since this had potential repercussions on LAL users